### PR TITLE
428 references post offer dashboard request references

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

This PR adds the request reference after the candidate accepted an offer.

## Guidance to review

1. Can you add more requests?
2. Can you send a request to a new reference?

## Link to Trello card

https://trello.com/c/LmawPklA/428-references-post-offer-dashboard

### CODE REVIEW

File: app/controllers/candidate_interface/new_references/request_reference/review_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module CandidateInterface
+  module NewReferences
+    class RequestReference::ReviewController < ReviewController
+      include RequestReferenceOfferDashboard
+      before_action :set_reference, :set_policy
+      before_action :verify_reference_can_be_requested, only: %i[feedback_requested]
+
+      def feedback_requested
```
Comment (carlosmartinez) [inline app/controllers/candidate_interface/new_references/request_reference/review_controller.rb:8]: would `request_feedback` be a more accurate name?

**Review by frankieroberto — COMMENTED** [review]
Tested this in the review app.

One bug: after accepting an offer, it's possible to add the name of a new reference but not their email address or relationship, and then use the back links to return to the main page, then click into that reference and send the request, even though it’s incomplete. We should remove the green 'Request reference' button if there any fields incomplete.

Design-wise, on the main page listing references, the second line saying "Request sent" with the date should say "Request sent on [date] - cancel request" with all the text being black and small, and 'cancel request' being a link to the confirm cancel page. See screenshots below:

## Design in prototype

Screenshot 2022-08-15 at 11 57 01

## Current design in pull request (with notes)

Screenshot 2022-08-15 at 11 52 52

File: app/controllers/candidate_interface/new_references/request_reference/review_controller.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/controllers/candidate_interface/new_references/request_reference/review_controller.rb]: Done! ♻️

**Comment by tomas-stefano** [comment]
@frankieroberto addressed the design change. The change "it's possible to add the name of a new reference but not their email address or relationship" is better to address as a separate PR because this is a deeper issue in the three places where you add references.

**Review by carlosmartinez — APPROVED** [review]

**Pull request merged**